### DISCUSSION TITLE
Modify Ant Media publish page such that it should merge audio from multiple devices and sent it to AMS ( 50 to  80 USD )

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5164] [Freelancing]
Modify Ant Media publish page such that it should merge audio from multiple device and sent it to AMS.
Comment if you are intrested.
![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/cff1638e-cb07-4cb0-a7fa-91fcc4657451)

https://github.com/ant-media/StreamApp/blob/master/src/main/webapp/samples/publish_webrtc.html

**Comment by 0xab3** [comment]
hi usamawizard,
i am interested.
i will try something and if it work i will get back to  you.

**Reply by asharamseervi** [reply]
Hey @notshahzad, Thanks for picking this up! How is it going? Would you like any more help on this?

And, if you are willing to collaborate on many tasks/opportunities like this, please register yourself here: https://docs.google.com/forms/d/e/1FAIpQLSdXmlp_ISap30nWv_zv2P4fp4AVjw1RLDhyJZ8_EsCtZUk5lw/viewform

**Comment by USAMAWIZARD** [comment]
@notshahzad  Thanks for Showing your interest , Sure I will get back to you soon after confirming with our customer.

**Reply by 0xab3** [reply]
Hi @USAMAWIZARD ,
I have implemented that feature to merge audio.

**Reply by USAMAWIZARD** [reply]
Thanks @notshahzad for developing this. I have informed that the feature has being developed. Will get back to you soon

**Reply by USAMAWIZARD** [reply]
can you please provide me your email or email me usama.tahseen@antmedia.io so that I can invitee you on a call for demo
